README.md

```diff
@@ -7,9 +7,10 @@
 
 ## Getting started
 
-- `npm install` in the root directory
-- `cd app` and `npm run dev` to start the local host server
+- `yarn` in the root directory to ensure you have the latest installed packages
```

Comment:
Thank you for updating the readme, this it helpful when users wish to run the local